sync-orders-to-bigQuery/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "moltin-google-bigquery",
+  "version": "0.1.0",
+  "description": "send order and order items to Google BigQuery",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "Matthew Foyle",
+  "license": "MIT",
+  "devDependencies": {
+    "@moltin/eslint-config": "^1.0.0",
+    "eslint": "^5.16.0",
+    "serverless-google-cloudfunctions": "*"
+  },
+  "dependencies": {
```

Comment:
Can we pin these dependencies to their versions? (Remove `^`)

Reply:
Done